checker/src/options.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+use clap::{App, AppSettings, Arg};
+use itertools::Itertools;
+use shellwords;
+
+/// Creates the clap::App metadata for argument parsing.
+fn make_options_parser<'a>() -> App<'a, 'a> {
+    // We could put this into lazy_static! with a Mutex around, but we really do not expect
+    // to construct this more then once per regular program run.
```

Comment:
I expect to see a comment when something is made static, since statics are usually evil, but there is no need to justify not making something static.